woocommerce/woocommerce
# Add customize store assembler hub onboarding tour

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #39713.

This PR adds Assembler onboarding tour.

Figma: Mrk6SERPZ4KrFHSjM0a8TK-fi-2631%3A253787


https://github.com/woocommerce/woocommerce/assets/4344253/90961500-68d7-4019-8d1b-b1f840ef6844





### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Use WP 6.3 
2. Install the `WooCommerce Beta Tester` plugin
3. Go to /wp-admin/tools.php?page=woocommerce-admin-test-helper and enable `customize-store` feature flag
4. Go to `WooCommerce > Home`
5. Click on `Customize your store` task 
6. Click on `Assembler Hub` button
7. Should see the Assembler Hub screen
8. Should see the onboarding welcome tour
9. Click on`Take a tour` button
10. Observe that resizing handlebar and the label are visible. Once  interacts with the handlebar, the label should disappear. 
11. Complete the tour
12. Reload the page and go to Assembler Hub again
13. Observe that the tour is not shown.
14. Delete the option `wp option delete woocommerce_customize_store_onboarding_tour_hidden`
15. Repeat steps 5-8 but click on skip or `x` icon to dismiss the tour
16. Confirm the tour is not shown again.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

Add customize store assembler hub onboarding tour

#### Comment 

</details>
